server/src/internalClusterTest/java/org/elasticsearch/index/mapper/DynamicMappingIT.java

```diff
@@ -213,6 +214,69 @@ public void onFailure(Exception e) {
         }
     }
 
+    public void testTotalFieldsLimitWithRunTimefields() {
```

Comment:
s/RunTime/Runtime ?

With this test you are verifying the behaviour for dynamic mapping updates, I think you also need a test for the scenario where runtime fields are mapped through put mappings API. There are already similar tests in MapperServiceTests.

Reply:
Great suggestion, addressed in 1ee42705cb6ad4c4c5b6083a33c042a1b8269912